package.json

```diff
@@ -4,8 +4,46 @@
   "description": "A Query Language and Runtime which can target any service.",
   "license": "MIT",
   "private": true,
-  "main": "index",
+  "main": "index.js",
   "module": "index.mjs",
+  "exports": {
+    "./execution/execute.js": {
+      "types": {
+        "import": "./execution/execute.js.d.mts",
+        "default": "./execution/execute.d.ts"
+      },
+      "import": "./execution/execute.js.mjs",
+      "module": "./execution/execute.mjs",
+      "default": "./execution/execute.js"
+    },
```

Comment:
These are just additional entry points. The folder-style entry points are generated during build.